Makefile

```diff
@@ -30,4 +31,4 @@ build:
 	docker build -t $(ARTIFACT) .
 
 run:
-	docker run -ti $(ARTIFACT)
+	docker run -i $(ARTIFACT)
```

Comment:
ditto `-i` question, though it's easier to imagine its utility here.

Reply:
Good question, I remove `t` but I will check why `i` is still there